services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None
 ) -> QuerySet:
```

Comment:
The return type annotation for `get_movies` should specify `QuerySet[Movie]` to indicate that the queryset contains `Movie` objects.